### ISSUE TITLE
Battery LED Light Indicator and Projected Battery Life Breaks After Updating to BIOS F7A0116 or Above

### ISSUE THREAD

**Issue by Pirangunter21** [ValveSoftware/SteamOS#1194]
#### Your system information

* Steam client version: Stable
* SteamOS version: 3.4.11
* Opted into Steam client beta?: [No] 
* Opted into SteamOS beta?: [No] 
* Have you checked for updates in Settings > System?: [**Yes**]

#### Please describe your issue in as much detail as possible:
After updating the SteamOS to version 3.4.11 (Stable), there's been issues with the LED indicator along with projected battery life. The issue is that when Steam Deck is fully charged, the LED light does not go off and it's bugging me because this has never happened before. I tried looking on the internet and saw some people having similar issue after updating, one in particular I found useful is to downgrade the BIOS.

https://github.com/ValveSoftware/SteamOS/issues/1138

Which ultimately I choose the latter and it works. I tried reverting to F7A0116 to see my theory is true and after back and forth the issue is persists on F7A0116 while F7A0110 works flawlessly. 

In order to fix the issue you have to follow this tutorial:
https://www.youtube.com/watch?v=GLvpBQX1pmI

Copy paste this to Terminal

Disclaimer: make sure to [download ](https://gitlab.com/evlaV/jupiter-hw-support/-/raw/0660b2a5a9df3bd97751fe79c55859e3b77aec7d/usr/share/jupiter_bios/F7A0110_sign.fd) the BIOS first and **BE SURE TO BACKUP YOUR BIOS** (you have been warned)
```
mkdir BIOS

cd BIOS

sudo /usr/share/jupiter_bios_updater/h2offt jupiter-$(sudo dmidecode -s bios-version)-bios-backup.bin -O

ls

sudo /usr/share/jupiter_bios_updater/h2offt ~/Downloads/F7A0110_sign.fd -U

Steam Deck will reboot to perform the downgrade.

cd BIOS

sudo dmidecode -s bios-version

sudo /usr/share/jupiter_bios_updater/h2offt jupiter-$(sudo dmidecode -s bios-version)-bios-backup.bin -O

```

As I mentioned before, the issue seems to be related to the bios version.

Hope this helps!

#### Steps for reproducing this issue:

1. Update SteamOS client to 3.4.11
2. Update BIOS to F7A0116  (this is automatically during 3.4.11 update)
3. Plug in Steam Deck and wait until it reaches 100% and see the LED light

**Comment by RodoMa92** [comment]
Yeah, the led not turning off is a BIOS bug introduced from 116, as you reported above (and even on the latest 119 the issue is still present). Not sure why it hasn't been fixed yet.

**Comment by waspennator** [comment]
> Yeah, the led not turning off is a BIOS bug introduced from 116, as you reported above (and even on the latest 119 the issue is still present). Not sure why it hasn't been fixed yet.

Feels like some issues seem to slip under the radar and only really get noticed after they make it to stable if it doesn't get noticed right away
